ingestion/src/metadata/ingestion/bulksink/metadata_usage.py

```diff
@@ -239,6 +230,48 @@ def __get_column_fqdn(
     def __get_table_entity(
         self, database_name: str, database_schema: str, table_name: str
     ) -> List[Table]:
+
+        # First try to find the data from the given db and schema
+        # Otherwise, pick it up from the table_name str
+        # Finally, try with upper case
+
+        split_table = table_name.split(".")
+
+        database_query, schema_query, table = (
+            [None] * (3 - len(split_table))
+        ) + split_table
+
+        table_entities = self.__fetch_table_entity(
```

Comment:
I think this should be named `table_entity` as, if I well understand, `__fetch_table_entity` return 1 entity.

Reply:
it actually should be the other way around to rename the function. it returns a list. Updating now